### PR TITLE
Nightly dev

### PR DESCRIPTION
**Pull request by prabhamuthu15**
### JIRA link (if applicable) ###
[DFPL-2191](https://tools.hmcts.net/jira/browse/DFPL-2191)

### Change description ###

Manage Document UI Test

**Does this PR introduce a breaking change?** (check one with "x")
[ ] Yes
[ x] No
```

### CODE REVIEW

File: playwright-e2e/pages/base-page.ts [hunk 1]
```diff
@@ -19,24 +19,25 @@ export class BasePage {
...
-    await this.goButton.dblclick();
-    await this.page.waitForTimeout(20000);
-    if (await  this.goButton.isVisible()) {
-       await this.goButton.click();
+      await expect(async () => {
+          await this.page.reload();
+          await this.nextStep.selectOption(eventName);
+          await this.goButton.click({clickCount:2,delay:300});
+          await expect(this.page.getByRole('button', { name: 'Continue' })).toBeVisible();
+        //  await expect(this.page.getByText(`${eventName}`,{exact:true})).toBeVisible();
```
Comment (jamesrferguson1) [inline playwright-e2e/pages/base-page.ts:33]: Do we need this commented out line?

File: playwright-e2e/pages/create-case.ts [hunk 2]
```diff
@@ -79,7 +79,7 @@ export class CreateCase extends BasePage{
     //await this.addApplicationTitle.isVisible();
 
     // This click timeout is here allow for ExUI loading spinner to finish
-    await this.viewHistory.click();
+    //await this.viewHistory.click();
```
Comment (jamesrferguson1) [inline playwright-e2e/pages/create-case.ts:82]: Guess if these lines are commented out and no longer needed we can just remove?

File: playwright-e2e/tests/manage-documents.spec.ts [hunk 3]
```diff
@@ -50,13 +58,152 @@ test.describe('Manage Documents', () => {
...
+        await updateCase(caseName, caseNumber, caseWithResSolicitor);
+        await giveAccessToCase(caseNumber, privateSolicitorOrgUser, '[SOLICITORA]');
+        await signInPage.visit();
+        await signInPage.login(newSwanseaLocalAuthorityUserOne.email, newSwanseaLocalAuthorityUserOne.password);
+        await signInPage.navigateTOCaseDetails(caseNumber);
+        await manageDocuments.gotoNextStep('Manage documents');
+        await manageDocuments.uploadDocuments('Position Statements');
+
+        // uploads documents
+       // await manageDocuments.uploadNewDocuments();
```
Comment (jamesrferguson1) [inline playwright-e2e/tests/manage-documents.spec.ts:73]: Can we just delete this comment?

File: playwright-e2e/pages/manage-documents.ts [hunk 4]
```diff
@@ -1,30 +1,103 @@
...
+        await this.documentType.selectOption(type);
+        await this.nonconfidentialRadioButton.check();
+        await this.isDocumentRelatedToCase.check();
+        await this.waitForAllUploadsToBeCompleted();
+        await this.clickContinue();
+        await this.checkYourAnsAndSubmit();
+        await this.addNew.check();
+        await this.no.check();
+    }
+    async uploadConfidentialDocuments(type: string) {
```
[the remark this answers is not in the document]
Comment (prabhamuthu15) [reply playwright-e2e/pages/manage-documents.ts]: Combined into single function

**Review by jamesrferguson1 — APPROVED** [review]

**Pull request merged**